### ISSUE TITLE
Font settings in config file are ignored

### ISSUE THREAD

**Issue by standaniels** [JGRennison/OpenTTD-patches#188]
## Version of OpenTTD
OpenTTD 1.10.3 + JGRpp 0.36.0 on MacOS 10.15.6 (19G2021).

## Expected result
Font face and alias settings are effect rendering text.

Screenshot of vanilla OpenTTD:
![image](https://user-images.githubusercontent.com/1199737/92921899-3225bf00-f435-11ea-9f8e-398e5026ccdb.png)

## Actual result
Font face and alias settings seem te be ignored while rendering text.

Screenshot of OpenTTD + JGRpp:
![image](https://user-images.githubusercontent.com/1199737/92921947-44076200-f435-11ea-8469-6af1c71fbda3.png)

## Steps to reproduce
Set the following font values:

```
small_font = SF Pro Text Medium
medium_font = SF Pro Text Medium
large_font = SF Pro Text Medium
mono_font = SF Mono Medium
small_size = 12
medium_size = 13
large_size = 18
mono_size = 12
small_aa = true
medium_aa = true
large_aa = true
mono_aa = true
```

[Full openttd.cfg file.](https://github.com/JGRennison/OpenTTD-patches/files/5208710/openttd.cfg.txt)

**Comment by JGRennison** [comment]
I'm not an Apple user and haven't made any Apple-specific changes in this area.
Upstream have made Apple-related changes recently.
It may be useful to test the most recent upstream nightly from here: https://www.openttd.org/downloads/openttd-nightlies/latest.html

**Comment by standaniels** [comment]
Thanks for your reply. I've tested the most recent upstream nightly (openttd-20200902-master-g53a3d940b1) and the issue still exists.

**Comment by James103** [comment]
Could you please search for an existing issue matching your description or link it here, or create a new issue in the main OpenTTD repo if no such issue exists there?

**Comment by standaniels** [comment]
@James103 I meant I've tried the latest nightly with jgrpp and the issue still exists. There's no issue with the main OpenTTD version.

**Comment by embeddedt** [comment]
If the problem is in the upstream nightly, then it's not a JGRPP issue.

**Comment by standaniels** [comment]
Alright, the problem exists both in the nightly build and in JGRPP, but not in the latest stable of OpenTTD. Does that mean that that JGRPP uses the nightly build and not the latest stable or how are these projects related?

**Comment by embeddedt** [comment]
JGRPP is based on nightly builds.

**Comment by standaniels** [comment]
That I didn't know. I'll open an issue in the upstream repo. Thank you for your help!

**Comment by ecolortest** [comment]
Yes, This issues is happen since OpenTTD changed compile configuring to CMake

**Comment by chaser92** [comment]
Hi,

Indeed, the fix is to add the freetype2 dependency 
in CMakeLists.txt:

there is:
```
if (NOT APPLE)
...
   find_package(Freetype)
```

Just move `find_package(Freetype)` before the conditional. Note that of course you need to have freetype installed, you can just do it through homebrew.

**Comment by ecolortest** [comment]
@chaser92 
![Screen Shot 2020-09-13 at 14 03 21](https://user-images.githubusercontent.com/34646054/93010675-fcfea580-f5c9-11ea-87dc-e6a6a11acf36.png)
Thanks, when i moved the font is now working, but im getting weird version string....

**Comment by standaniels** [comment]
Thanks @chaser92, that solved the issue.

**Comment by JGRennison** [comment]
Fixed in v0.37.0
